### PR TITLE
Adding common types from attr

### PR DESCRIPTION
**Pull request by paganotoni**
@markbates this PR adds common types to fizz translators, i started with Postgres and would love to know im moving in the right direction before going forward.

### CODE REVIEW

**Comment by markbates** [comment]
👍

**Comment by paganotoni** [comment]
@gobuffalo/pop-managers this one is ready for review, please LMK if its looking good for you.

File: database.yml [hunk 1]
```diff
@@ -4,7 +4,10 @@ mysql:
   host: {{ envOr "MYSQL_HOST" "127.0.0.1"  }}
   port: {{ envOr "MYSQL_PORT" "3306"  }}
   user: {{ envOr "MYSQL_USER"  "root"  }}
-  password: {{ envOr "MYSQL_PASSWORD"  "root"  }}
+  password: {{ envOr "MYSQL_PASSWORD" "root"  }}
+  options:
```
Comment (stanislas-m) [inline database.yml]: These options don't exist.

**Comment by paganotoni** [comment]
@stanislas-m all set, removed these options.

**Comment by stanislas-m** [comment]
Thanks @paganotoni!

**Pull request merged**